pkg/agent/plugin/workloadattestor/docker/docker_test.go

```diff
@@ -200,6 +203,42 @@ func TestDockerError(t *testing.T) {
 	require.Nil(t, res)
 }
 
+func TestDockerErrorRetries(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	mockDocker := NewMockDockerClient(mockCtrl)
+	mockFS := filesystem_mock.NewMockFileSystem(mockCtrl)
+	mockClock := clock.NewMock(t)
+
+	p := New()
+	p.docker = mockDocker
+	p.fs = mockFS
+	p.retryer.clock = mockClock
+
+	cgroupFile, cleanup := newTestFile(t, "1:foo:/bar")
+	defer cleanup()
+	mockFS.EXPECT().Open("/proc/123/cgroup").Return(os.Open(cgroupFile))
+	for i := 0; i < 4; i++ {
+		mockDocker.EXPECT().
```

Comment:
It doesn't look like you are using `i`. I believe there is a `.Times(4)` you could use.